### DISCUSSION TITLE
Using custom dataset for segmentation

### DISCUSSION THREAD

**Discussion by Holmes2002** [yoshitomo-matsubara/torchdistill#258] [Q&A]
Just upload images and labeled images according to form of pascal voc. Edit your paths, set download = False in .yaml. If out of memory in training process, reduce min shape and max shape.

I think it is all for segmentation task. Do i miss something ?

**Comment by Holmes2002** [comment]
I also have question that resnet 18 is available or not ? I think resnet 50 is quite weight

**Comment by yoshitomo-matsubara** [comment]
Hi @Holmes2002 

> I think it is all for segmentation task. Do i miss something ?

I need more details. Are you seeing only out of memory error?

> I also have question that resnet 18 is available or not ? I think resnet 50 is quite weight

Same here, I need more details. Are you asking about DeepLabv3 with ResNet-18 instead of ResNet-50?

P.S. Please reply to this thread

**Reply by Holmes2002** [reply]
would u like to  add ? I think it is enough. I can run model easily

**Reply by yoshitomo-matsubara** [reply]
Sorry, I cannot understand the context

Answer my questions above and/or clarify your question
